src/coord/src/catalog/builtin.rs

```diff
@@ -1044,7 +1040,14 @@ pub const PG_TYPE: BuiltinView = BuiltinView {
     false::pg_catalog.bool AS typnotnull,
     0::pg_catalog.oid AS typbasetype
 FROM mz_catalog.mz_types
-JOIN mz_catalog.mz_schemas ON mz_schemas.id = mz_types.schema_id",
+JOIN mz_catalog.mz_schemas ON mz_schemas.id = mz_types.schema_id
+JOIN (
+    SELECT type_id, 'a' AS typtype FROM mz_catalog.mz_array_types
+        UNION SELECT type_id, 'b' FROM mz_catalog.mz_base_types
+        UNION SELECT type_id, 'l' FROM mz_catalog.mz_list_types
+        UNION SELECT type_id, 'm' FROM mz_catalog.mz_map_types
+    ) AS t
+    ON mz_types.id = t.type_id",
```

Comment:
I think this line and the line above it want to be unindented to the level of `JOIN`.